Features/Lookuptable/testPages/data/lookup_table_page.py

```diff
@@ -0,0 +1,771 @@
+        excel = ExcelManager(download_path_1)
+        excel.delete_row(tablename, 2)
+        self.replace_existing_table(download_path_1)
+        self.view_lookup_table(tablename)
+        row_count_after = self.rowCount_table(tablename)
+        assert row_count_before > row_count_after
+        print("Row got successfully Deleted")
+
+    def rowCount_table(self, tablename):
+        self.view_lookup_table(tablename)
+        text = self.get_text(self.rowcount)
+        rowcount = text[18:20].strip()
+        return rowcount
+
+    def update_delete_field(self,download_path,tablename):
+        excel = ExcelManager(download_path)
+        excel.upload_to_path(tablename, UserData.data_list)
+        self.err_upload(download_path)
+        self.view_lookup_table(tablename)
+        row_count_before = self.rowCount_table(tablename)
+        download_path_1 = self.latest_download_file()
+        excel = ExcelManager(download_path_1)
+        excel.write_excel_data(tablename, 3, 2, 'Y')
+        self.replace_existing_table(download_path_1)
+        self.view_lookup_table(tablename)
+        row_count_after = self.rowCount_table(tablename)
+        assert row_count_before > row_count_after
+        print("Row got successfully Deleted")
+
+    def attribute_2(self,download_path,tablename):
+        download_path = self.latest_download_file()
+        excel = ExcelManager(download_path)
+        excel.write_excel_data("types", 1, 4, 'field 1')
+        excel.write_excel_data("types", 2, 4, 'C1')
+        excel.write_excel_data("types", 1, 5, 'field 2')
+        excel.write_excel_data("types", 2, 5, 'C2')
+        excel.write_excel_data(tablename, 1, 3, 'field:C1')
+        excel.write_excel_data(tablename, 1, 4, 'field:C2')
+        excel.write_data(tablename, UserData.new_datalist)
+        self.err_upload(download_path)
+        self.view_lookup_table(tablename)
+        UIrows = self.rowCount_table(tablename)
+        self.download1()
+        excelrows = excel.row_size(tablename)
+        assert int(UIrows) == (excelrows - 1)
+
+    def bulkupload_1(self,tablenames, n, upload_path):
+        before = ""
+        after =""
+        for i in range (0,n):
+            tablename = str.split(tablenames,":")[i]
+            row_count_before = self.rowCount_table(tablename)
+            before = before +","+ row_count_before
+        before=str.split(before,",",1)[1].strip()
+        self.err_upload(upload_path)
+        for i in range (0,n):
+            tablename = str.split(tablenames,":")[i]
+            row_count_After = self.rowCount_table(tablename)
+            after = after +","+  row_count_After
+        after = str.split(after, ",",1)[1].strip()
+        for i in range (0,n):
+            b_row = str.split(before,',')[0]
+            a_row = str.split(after, ',')[0]
+            print(b_row , a_row)
+            assert b_row < a_row
+
+
+    def delete_field_columns(self,downloadpath,tablename):
```

Comment:
remove download_path from parameters, as you are using it after defining inside the function.